keel-actuator/src/main/kotlin/com/netflix/spinnaker/keel/actuation/ResourceCheckHealthIndicator.kt

```diff
@@ -0,0 +1,36 @@
+package com.netflix.spinnaker.keel.actuation
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.boot.actuate.health.AbstractHealthIndicator
+import org.springframework.boot.actuate.health.Health
+import org.springframework.context.event.EventListener
+import org.springframework.stereotype.Component
+import java.time.Clock
+import java.time.Duration
+import java.time.Instant
+import java.time.format.DateTimeFormatter.ISO_LOCAL_DATE_TIME
+
+@Component
+class ResourceCheckHealthIndicator(
+  private val clock: Clock,
+  @Value("\${keel.health.resource-check.threshold:PT1M}") private val healthyThreshold: Duration
```

Comment:
We'll start failing health checks if the scheduled check does not happen at least once per minute.